docs/getting-started/README.md

```diff
@@ -60,14 +60,14 @@ npm install
 # Or `yarn install`, if you prefer. We'll stick to npm commands.
 ```
 
-Before starting up you'll need to create an admin-level user so that you can to log in. After running the following command, Apostrophe will ask you to enter a password for this user.
+Before starting up you'll need to create an admin-level user so that you can log in. After running the following command, Apostrophe will ask you to enter a password for this user.
 
 ```bash
-node app @apostrophecms/user:add my-user
+node app @apostrophecms/user:add my-user admin
 # Replace `my-user` with the name you want for your first user.
 ```
 
-Start the site with `npm run dev`. The app will then watch for changes, then restart the app and refresh the browser when it detects any. You can log in with the username and password you created at [http://localhost:3000/login](http://localhost:3000/login).
+Start the site with `npm run dev`. The app will then watch for changes in client-side code, then restart the app and refresh the browser when it detects any. You can log in with the username and password you created at [http://localhost:3000/login](http://localhost:3000/login).
 
 ::: tip
 If you are starting the site in a production or do not want the process to watch for changes, start the site with `node app.js`.
```

Comment:
"In a production environment"

There's more to say about how they should run it in production but this is OK for now.